tests/test_client.py

```diff
@@ -46,3 +98,31 @@ def test_get_projects(self, mock_get):
         self.assertEqual(len(projects), 2)
         for project in projects:
             assert isinstance(project, Project)
+
+    @mock.patch('testrail.api.requests.get')
```

Comment:
Bit of a nitpick but could you separate each assert into a method similar to the other tests.